pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -8,7 +8,7 @@ let
   gitBase = lib.makeOverridable (import ./git) {
     inherit fetchurl stdenv curl openssl zlib expat perl python gettext gnugrep
       asciidoc xmlto docbook2x docbook_xsl docbook_xml_dtd_45 libxslt cpio tcl
-      tk makeWrapper subversionClient gzip;
+      tk makeWrapper subversionClient gzip libiconvOrNull;
```

Comment:
Note that `libiconvOrNull` will probably go away soon: https://github.com/NixOS/nixpkgs/pull/6166